e2e-tests/trade-finance-manager/cypress/integration/pages/amendments/amendmentsRequestPage.js

```diff
@@ -0,0 +1,16 @@
+const amendmentsRequestPage = {
+  amendmentRequestHeading: () => cy.get('[data-cy="amendment-request-heading"]'),
+  amendmentRequestHint: () => cy.get('[data-cy="amendments-request-hint"]'),
+  amendmentRequestDayInput: () => cy.get('[data-cy="amendment-request-date-day"]'),
+  amendmentRequestMonthInput: () => cy.get('[data-cy="amendment-request-date-month"]'),
+  amendmentRequestYearInput: () => cy.get('[data-cy="amendment-request-date-year"]'),
+
+  errorSummary: () => cy.get('[data-cy="error-summary"]'),
+  errorMessage: () => cy.get('[data-cy="amendment-request-date-error"]'),
+
+  continueButton: () => cy.get('[data-cy="continue-button"]'),
+  cancelLink: () => cy.get('[data-cy="cancel-button"]'),
```

Comment:
Can we please use similar naming convention as it is being used elsewhere

* `cancel-button` => `amendment-cancel-button`
* `continue-button` => `amendment-continute-button`
* `error-summary` => `amendment-error-summary`

Reply:
this file doesn't exist anymore actually. I've removed it